pom.xml

```diff
@@ -95,7 +95,7 @@
               <configuration>
                 <rules>
                   <requireJavaVersion>
-                    <version>[1.7.0-45,1.8)</version>
+                    <version>[1.7.0-45,1.8.0-31,1.8)</version>
```

Comment:
wasn't really keen on allowing java8 builds of the project.
is this required in this pull request?